### DISCUSSION TITLE
CMI requests

### DISCUSSION THREAD

**Discussion by Th3v0x** [Zrips/CMI#7902] [Enhancement]
The plugin is crazy but I have some recommendations that maybe you could consider. 
You could add something like a sort of geoip to locate from where each user connects to the server.
Also would be nice the /lay command that the functionality is for the user to lie down.
Add support with MiniMessage to color the configuration messages for example <rainbow><obfuscated><strikethrough><strikethrough><strikethrough><color:#55FF55><gradient:blue:red><b>.
Thanks for reading me and keeping me in mind.

**Comment by mrfloris** [comment]
<img width="676" alt="Screen-Shot 2023-06-20 at 18 39 59" src="https://github.com/Zrips/CMI/assets/28841349/3051aac9-63ee-479c-a9e4-c60bc89e1520">


Sorry to make you jump through hoops, but issues is for bugs, and discussions is for requests; 
https://github.com/Zrips/CMI/discussions

**Comment by CloudeLecaw** [comment]
hmm duplicate of https://github.com/Zrips/CMI/discussions/7901
please close this one...
